### DISCUSSION TITLE
Make Periodic Tasks timezone-aware

### DISCUSSION THREAD

**Discussion by ashleyheath** [procrastinate-org/procrastinate#1029] [Ideas]
I have a use case for using procrastinate to schedule different periodic tasks to run according to different timezone offsets (e.g. have one periodic task run at midnight UTC and another run at midnight in Europe/London.

Under the hood, it looks like croniter supports this (`set_current` will set the `tzinfo` if the value passed in is a `datetime`).

Have started a skeleton of a PR but just wanted to check the project is happy to accept the change (and if there are any things I should keep in mind when doing the implementation).

Thanks!

**Comment by ewjoachim** [comment]
That could be interesting, it doesn't seem like a huge development, and it shouldn't change the behavior in the default case.

Go for it :)
